### ISSUE TITLE
Discord does not open because of a very specific configuration

### ISSUE THREAD

**Issue by la-manti** [flathub/com.discordapp.Discord#113]
Hi. My flathub remote is of the "user" type, located at ~/.local/share/flatpak/ and their app data, by default, is stored at ~/.var. But I decided to create a symbolic link from **.var** and **flatpak** directories, to a different location where I keep personal data in a NFTS partition, shared with Windows. 
But there is a problem: because of **.var** is located there, Discord doesn't open. When I open through terminal, the following message is shown:

```
Discord 0.0.12
[fake-sandbox: zypak-sandbox] No data could be read (host died?)
[fake-sandbox: zypak-sandbox] Quitting Zygote...
/app/bin/discord: line 8:     4 Trace/breakpoint trap   (core dumped) env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/discord/Discord "$@"
```

In other words, the location of **.var** affects the opening of Discord for some reason I can't understand. This behaviour doesn't happen when I create a symbolic link inside home directory. Ex: **~/.var** pointed at **~/var2**. But when I create a symbolic link to my NFTS partition, this error happens. 
Note: it doesn't affect other apps. Every app opens normally inside Flatpak.

**Comment by la-manti** [comment]
> it is likely not following the symbolic link to your nfts partition.
So... I can think the developers from this version of Discord can fix this as far as no app from mine have the same issue.

**Comment by Fuseteam** [comment]
the only way to know that is to see if deb "version" has the same issue, this isn't a version of discord, its just a way to package it

**Comment by la-manti** [comment]
> the only way to know that is to see if deb "version" has the same issue, this isn't a version of discord, its just a way to package it

The native client works normally in Manjaro.

**Comment by Fuseteam** [comment]
i have used the native client on ubuntu so that's definitely not true

**Comment by la-manti** [comment]
> 
> 
> i have used the native client on ubuntu so that's definitely not true

What?

**Comment by Fuseteam** [comment]
i have installed and used the deb package of discord on ubuntu sooo......wait i think i misread, i read your comment as "the native client only works in majaro"
so if it works normally on manjaro does it also work normally with the specific configuration you mentioned here?

**Comment by la-manti** [comment]
> so if it works normally on manjaro does it also work normally with the specific configuration you mentioned here?

Yes.

**Comment by la-manti** [comment]
Just to know: I created a symbolic link in **discord** directory in **~/.config**. There was no error or issues.

**Comment by refi64** [comment]
What paths *exactly* are you symlinking onto the NTFS partition?

**Comment by Fuseteam** [comment]
hmmm

**Comment by guihkx** [comment]
Closing due to lack of follow-up from reporter.
